content/he/faq.md

```diff
@@ -1,34 +1,34 @@
-# FAQ
+<div dir="rtl" lang="he">
 
-## What is io.js?
+# שאלות נפוצות
 
-[io.js](https://github.com/iojs/io.js) is a JavaScript platform built on [Chrome's V8 runtime](http://code.google.com/p/v8/). This project began as a fork of [Joyent's Node.js™](https://nodejs.org/) and is compatible with the [npm](https://www.npmjs.org/) ecosystem.
+## מה זה io.js?
 
-Why? io.js aims to provide faster and predictable release cycles. It currently merges in the latest language, API and performance improvements to V8 while also updating libuv and other base libraries.
+io.js היא פלטפורמת ג'אווהסקריפט הבנוייה על [מנוע V8 של כרום](http://code.google.com/p/v8/). הפרוייקט התחיל כפיצול מ [Node.js™ של חברת Joynet](https://nodejs.org/) והוא עובד היטב עם מערכת ה[npm](https://www.npmjs.org/)
 
-This project aims to continue development of io.js under an "[open governance model](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme)" as opposed to corporate stewardship.
+המטרה של io.js היא לספק שחרורים בקצב מהיר וצפוי יותר.  כרגע נמצאים בו השיפורים העדכניים לשפה, הAPI והביצועים של V8, בנוסף, אנו מעדכנים את libuv וספריות בסיס אחרות.
 
-## Version 1.0.x?
+מטרת הפרוייקט היא להמשיך את הפיתוח של io.js תחת מודל "[ממשל פתוח](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme)", בניגוד לשליטה ע"י קואופרטיב או חברה.
 
-io.js has moved to [Semver](http://semver.org/) and the changes between Node.js™ 0.10 and io.js 1.0.0 were significant enough
-to warrant a major version increment.
 
-Our [CHANGELOG](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md) for v1.x provides a [summary of changes from Node.js v0.10.35 to io.js v1.0.x](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md#summary-of-changes-from-nodejs-v01035-to-iojs-v100).
+## גירסה 1.0.x?
```

Comment:
I don't think this line renders quite correctly?
![screen shot 2015-03-04 at 1 21 54 pm](https://cloud.githubusercontent.com/assets/1093990/6490060/7c7b4490-c271-11e4-8d08-d4308b00817d.png)